cmd/root.go

```diff
@@ -40,6 +40,7 @@ func init() {
 	RootCmd.AddCommand(NewUpdateCmd())
 	RootCmd.AddCommand(NewInitCmd())
 
+	log.SetOutput(os.Stderr)
```

Comment:
Are you sure we should do this?

Reply:
Since we want to allow piping of the klepto output we should not use stdout for logs/errors.
Also see https://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout